nixos/modules/system/activation/switch-to-configuration.pl

```diff
@@ -470,13 +471,13 @@ sub handleModifiedUnit {
 my %unitsToFilter; # units not shown
 
 %unitsToStart = map { $_ => 1 }
-    split('\n', read_file($startListFile, err_mode => 'quiet') // "");
+    split("\n", read_file($startListFile, err_mode => "quiet") // "");
 
 %unitsToRestart = map { $_ => 1 }
-    split('\n', read_file($restartListFile, err_mode => 'quiet') // "");
+    split("\n", read_file($restartListFile, err_mode => "quiet") // "");
 
 %unitsToReload = map { $_ => 1 }
-    split('\n', read_file($reloadListFile, err_mode => 'quiet') // "");
+    split("\n", read_file($reloadListFile, err_mode => "quiet") // "");
```

Comment:
Careful, `"\n" != '\n'`.

Reply:
A later commit does exactly that ;)

Reply:
Heh, I don't know how I missed it

Reply:
Don't worry about it, I was confused as well because I couldn't find `'\n'` in the file :D